packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -935,7 +933,7 @@ export class DatabaseInstanceReadReplica extends DatabaseInstanceNew implements
     });
 
     this.connections = new ec2.Connections({
-      securityGroups: [this.securityGroup],
+      securityGroups: this.securityGroups,
       defaultPort: ec2.Port.tcp(this.instanceEndpoint.port)
```

Comment:
I see this code:

```ts
    this.connections = new ec2.Connections({
      securityGroups: [this.securityGroup],
      securityGroups: this.securityGroups,
      defaultPort: ec2.Port.tcp(this.instanceEndpoint.port)
    });
```

is repeated in `DatabaseInstance`, `DatabaseInstanceFromSnapshot` and `DatabaseInstanceReadReplica`. Is there any way we can reduce this duplication?